src/main/java/org/isf/lab/gui/elements/ExamComboBox.java

```diff
@@ -21,14 +21,14 @@
  */
 package org.isf.lab.gui.elements;
 
+import java.util.List;
+import java.util.Optional;
+import javax.swing.*;
```

Comment:
Project standards are to enumerate all the classes and not to use "*"; thanks

Reply:
done